packages/service-provider-server/src/cli-service-provider.integration.spec.ts

```diff
@@ -798,7 +798,7 @@ describe('CliServiceProvider [integration]', function () {
 
   describe('#getURI', function () {
     it('returns the current URI', function () {
-      expect(serviceProvider.getURI()).to.equal(connectionString + '/');
+      expect(serviceProvider.getURI()).to.equal(connectionString);
```

Comment:
There was no change in ServiceProvider, will this not have a trailing slash anymore?

Reply:
It’s not the `ServiceProvider` method that changed here, it’s that `connectionString` variable in the test itself now includes the trailing slash (which I think is technically required by the connection string spec, so I’d consider it a bug fix either way)

Reply:
Ahh got it. Thanks for explaining 👍